src/components/data_display/routes.ts

```diff
@@ -97,6 +101,7 @@ const routeDataViewDetail = async (request: any, reply: any) => {
   const dataViewCode = camelCase(request.params.dataViewCode)
   const recordId = Number(request.params.id)
   const { userId, orgId, permissionNames } = await getPermissionNamesFromJWT(request)
+  if (request.auth.isAdmin) permissionNames.push(LOOKUP_TABLE_PERMISSION_NAME)
```

Comment:
Note that we only add this for the "table", "detail" and "filter list" routes, not the "list" route, as we don't want lookup table definitions appearing in the "Database" menu.

Reply:
I have no idea what do you mean... hahaha

But good that LookupTables don't show there.